metsim/metsim.py

```diff
@@ -191,25 +231,25 @@ def read_netcdf(self, fpath:str) -> xr.Dataset:
         ds['day_of_year'] = (['time'], dates.dayofyear)
 
         # Update the configuration
-        configuration.update({"n_days" : n_days})
+        MetSim.params.update({"n_days" : n_days})
 
         #TODO: If the domain file and input file don't have the same grid
         #      structure this will have to be implemented
         #in_lats, in_lons = np.meshgrid(self.lats, self.lons)
         #out_lats, out_lons = np.meshgrid(ds.lat, ds.lon)
         #self.elev = basemap.interp(self.elev, self.lats, self.lons, out_lats, out_lons, order=1)
-        return ds 
-    
-    
+        return ds
+
+
     def read(self, fpath:str) -> xr.Dataset:
         """
-        Dispatch to the right function based on the file extension 
+        Dispatch to the right function based on the file extension
         """
         ext_to_fun = {
                 '.bin'   : self.read_binary,
                 '.nc'    : self.read_netcdf,
```

Comment:
I suppose the default is NetCDF 4, since we use the `netCDF4` library.  It can handle version 3 as well, if I recall correctly.